### ISSUE TITLE
Two axis quaternion constructor with vectors facing away

### ISSUE THREAD

**Issue by gaborpapp** [g-truc/glm#469]
It seems that the quaternion constructor does not handle the special case when the two vectors are facing away from each other. Using this code:

```
glm::quat A = glm::quat(glm::vec3(0, 1, 0), glm::vec3(0, -1, 0));
glm::quat B = glm::rotation(glm::vec3(0, 1, 0), glm::vec3(0, -1, 0));
```

I get different results `[0, 0, 0, 1]` and `[-1, 0, 0, 0]`. `glm::rotation` gives the expected one.

**Comment by Groovounet** [comment]
So this feature is actually already supported ... :D

Thanks for reporting,
Christophe

**Comment by Groovounet** [comment]
This issue is fixed in master branch for GLM 0.9.9 release.

Thanks for contributing,
Christophe
